### ISSUE TITLE
Line2D/Polygon2D rotates wrong direction using vertex shader with batching enabled

### ISSUE THREAD

**Issue by briansemrau** [godotengine/godot#48452]
**Godot version:**

3.x d08666f999f3912e2528994a7dad42eada236b5f 

**OS/device including version:**

Windows 10
GLES3

**Issue description:**

This appears to be the same issue as https://github.com/godotengine/godot/issues/46895

Adding the following shader to a Line2D or Polygon2D makes them rotate the wrong way:
```glsl
shader_type canvas_item;
render_mode blend_mix;

void vertex() {
    VERTEX;
}
```

With batching OFF:
![image](https://user-images.githubusercontent.com/6376721/117019999-fe4d2180-acc3-11eb-8edc-c04ead7699b7.png)

With batching ON:
![image](https://user-images.githubusercontent.com/6376721/117020098-1de44a00-acc4-11eb-8f70-bda545e731c8.png)

**Steps to reproduce:**
See above.

**Minimal reproduction project:**

[BatchingRotationReprod.zip](https://github.com/godotengine/godot/files/6421752/BatchingRotationReprod.zip)

**Comment by Calinou** [comment]
cc @lawnjelly

**Comment by briansemrau** [comment]
FYI I'm working on the fix right now.

**Comment by akien-mga** [comment]
Fixed by #48457.
